src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -104,10 +104,12 @@ def get(self, fs_id):
         return self.fs_status(fs_id)
 
     @RESTController.Resource('GET')
-    def clients(self, fs_id):
+    def clients(self, fs_id, **kwargs):
+        suppress_client_ls_errors = kwargs.pop('suppress_client_ls_errors',
+                                               'True')
         fs_id = self.fs_id_to_int(fs_id)
 
-        return self._clients(fs_id)
+        return self._clients(fs_id, suppress_client_ls_errors)
```

Comment:
return self._clients(fs_id, suppress_client_ls_errors=suppress_client_ls_errors)

Reply:
ah, thanks @YiteGu, fixed and updated :)